dialogflow/cloud-client/src/main/java/com/example/dialogflow/DetectIntentKnowledge.java

```diff
@@ -104,48 +101,14 @@ public static void detectIntentKnowledge(
           System.out.format(" - Answer: '%s'\n", answer.getAnswer());
           System.out.format(" - Confidence: '%s'\n", answer.getMatchConfidence());
         }
+
+        allKnowledgeAnswers.add(Tuple.of(text, sessionsClient
```

Comment:
Another suggestion on the new sample guidelines is to explicitly define variables ahead of time, and avoid multiple nesting in function calls.

Reply:
Updated to break out nested calls.